src/qt/bitcoinstrings.cpp

```diff
@@ -162,7 +162,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", "Rebuild block chain index from current blk000
 QT_TRANSLATE_NOOP("bitcoin-core", "Rescan the block chain for missing wallet transactions"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Rescanning..."),
 QT_TRANSLATE_NOOP("bitcoin-core", "Run in the background as a daemon and accept commands"),
-QT_TRANSLATE_NOOP("bitcoin-core", "SSL options: (see the Bitcoin Wiki for SSL setup instructions)"),
```

Comment:
Are you sure that wiki page exists for Peercoin? Maybe this should keep the referral to the Bitcoin Wiki?

Reply:
http://wiki.peercointalk.org/index.php?title=Main_Page

Reply:
Can we merge this first and then change it back?

Reply:
why not just update the PR? you can just make the change and push it to your branch, the PR will update automatically.

Reply:
Ok. Done.